charts/kafka-operator/values.yaml

```diff
@@ -72,3 +72,11 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## Whether to create the LogForwarding or not.
+## For enabling LogForwarding feature, set the logForward.create to true.
+logForward:
```

Comment:
logForward as a parameter seems off here - can you remove it and use `additionalSidecars` instead to check?